com.sublimetext.three.yaml

```diff
@@ -10,8 +10,10 @@ sdk: org.freedesktop.Sdk
 separate-locales: false
 finish-args:
   - --share=ipc
-  - --socket=x11
```

Comment:
You can leave this to support older `flatpak` versions.

Reply:
We could, but I suppose the fallback exists in order to use wayland when available for security.

When was the fallback option introduced?

Reply:
If you have both flags it still behaves with the fallback behavior. I don't remember the exact versions offhand.

Reply:
Ah, okay, I see that now from the command reference (http://docs.flatpak.org/en/latest/flatpak-command-reference.html)

I got the info to use fallback-x11 from http://docs.flatpak.org/en/latest/sandbox-permissions.html, which does not mention that x11 + wayland does the same thing.

It is not relevant to this package, but maybe we should look into getting the fallback-x11 deprecated, as it seems to not make much sense to have...

Reply:
You misunderstood, you still need *both* flags.

Reply:
Ah, okay, I thought "if you have both flags" referred to x11 + wayland, not x11 + fallback-x11.

Fixed.